x-pack/plugins/osquery/cypress/support/e2e.ts

```diff
@@ -36,12 +38,22 @@ registerCypressGrep();
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace Cypress {
+    interface SuiteConfigOverrides {
+      env?: {
+        ftrConfig: SecuritySolutionDescribeBlockFtrConfig;
+      };
+      tags?: string | string[];
+    }
+
```

Comment:
do you think this interface definition is needed here?

`env` comes from `x-pack/test_serverless/functional/test_suites/security/cypress/cypress.d.ts`,
`tags` comes from `node_modules/@cypress/grep/src/index.d.ts`,
so maybe there's no reason for redefined them here. what do you think?